cmd/gitops/add/clusters/cmd.go

```diff
@@ -96,6 +99,32 @@ func getClusterCmdRunE(endpoint *string, client *resty.Client) func(*cobra.Comma
 			}
 		}
 
+		var profilesValues []capi.ProfileValues
+
+		for _, p := range flags.Profiles {
+			valuesPairs := strings.Split(p, ",")
+			profileMap := make(map[string]string)
+
+			for _, pair := range valuesPairs {
+				kv := strings.Split(pair, "=")
+				profileMap[kv[0]] = kv[1]
+			}
+
+			profileJson, err := json.Marshal(profileMap)
+			if err != nil {
+				return err
+			}
+
+			var profileValues capi.ProfileValues
+
+			err = json.Unmarshal(profileJson, &profileValues)
+			if err != nil {
+				return err
+			}
+
+			profilesValues = append(profilesValues, profileValues)
+		}
```

Comment:
Maybe pull this out into a `parseProfileFlags(flags.Profiles)` and add a few tests around failure handling? wdyt?

Reply:
Makes sense to me 🙂 👍